http/src/request/channel/message/update_message.rs

```diff
@@ -123,6 +125,28 @@ impl<'a> UpdateMessage<'a> {
         }
     }
 
+    /// Specify an attachment to keep.
+    ///
+    /// If called, all unspecified attachments will be removed from the message.
+    /// If not called, all attachments will be kept.
+    pub fn attachment(mut self, attachment: Attachment) -> Self {
```

Comment:
Should this be "attachment" or "file"? What's the difference in the terminology here compared to `ExecuteWebhook`?

Reply:
The reason `CreateMessage::attachment` was deprecated was to unify the `file` and `files` methods in functionality. They are now all for uploading files. `{UpdateMessage,UpdateWebhookMessage}::attachment{,s}`  are for specifying which attachments to keep, i. e. Which attachments **not** to delete.